src/chickens_everest.c

```diff
@@ -36,6 +36,43 @@ static void init_common_everest(void)
     reg_clr(s3_0_c15_c2_4, BIT(0) | BIT(1) | BIT(16) | BIT(17) | BIT(18) | BIT(22));
 }
 
+void init_t8122_everest(int rev)
```

Comment:
There is a `init_common_everest()` function, it should be used.

`init_t6030_everest()` already have the duplication problem so maybe modify that one to use init_common_everest() as well.

in `init_common_everest()`, s3_0_c15_c2_3 and s3_0_c15_c2_4 appears to be HID30 and HID31 respectively